.github/workflows/test.yml

```diff
@@ -142,14 +142,22 @@ jobs:
       - run:  mkdir -p ${{env.TEST_RESULTS}}
```

Comment:
The changes in this file would be removed when merging in the feature branch. This is because the most recently released version of consul does not have peering.

Reply:
Yup agreed, we'd not want to cut a release with all this unless there's something for Consul also plus we'll want to address TODOs and unit tests (for peering dialer) on the feature branch `peering` before merging to main.